v1Tov2Migrations/migrate_trees.js

```diff
@@ -9,7 +9,14 @@ const createTree = require('./helper/createTree');
 async function migrate() {
   try {
     // filter out trees with invalid planter_id
-    const base_query_string = `select * from treetracker.capture where tree_id is null and length(note) > 4`; // length(note) > 4 is temporary
+    const base_query_string = `
+    select tc.* from treetracker.capture  tc
+    join trees t
+    on tc.reference_id = t.id
+    where tree_id is null and t.planting_organization_id = 1642
```

Comment:
Using the 1642 as the source of new tree is safer, Seba will be responsible for choosing the new tree, also, 1642 so current the map for freetown to show the new tree.